cognite/client/_api/data_modeling/instances.py

```diff
@@ -213,29 +214,29 @@ def retrieve(
         self,
         nodes: NodeId | Sequence[NodeId] | tuple[str, str] | Sequence[tuple[str, str]] | None = None,
         edges: EdgeId | Sequence[EdgeId] | tuple[str, str] | Sequence[tuple[str, str]] | None = None,
-        sources: list[ViewId] | ViewId | None = None,
+        sources: list[ViewIdentifier] | ViewIdentifier | None = None,  # list[ViewId] | ViewId | None = None,
```

Comment:
Remove comment, and use `Sequence` on input.
```suggestion
        sources: Sequence[ViewIdentifier] | ViewIdentifier | None = None,
```

Reply:
Fixed in new commit